test/testCombination.jl

```diff
@@ -1,58 +1,31 @@
 @testset "Combination tests" begin
```

Comment:
What does this test?

Reply:
It tests combination for two drugs and sets their progression rates manually as you said in Slack. In drugA, g1 prog. rate in control is 1.0 and decreases to 0.8 for other concentrations; in drugB g1 prog. rate in control is 1.0 and decreases to 0.5. After we scale and calculate the Bliss and rescale back, the combined effect must be between 0.3 and 0.5 as the assert tests.
I made two more cases where the effect in end concentration is higher than control in both drugs we combine (like what we have in some cases) and when one is increasing and one is decreasing.